blocks/product-details/product-details.js

```diff
@@ -228,6 +230,8 @@ export default async function decorate(block) {
         setMetaTags(product);
         document.title = product.name;
       }
+      // Ratings from gql extensibility
+      $ratings.append(product.ratings.average, product.ratings.total);
```

Comment:
For some reason, `product.ratings` is undefined, although the initializer has the object in the response, which means the query was successfully augmented.

I'm probably missing something obvious, like a transform or something.

<img width="818" alt="image" src="https://github.com/user-attachments/assets/65f857a4-032b-42d3-b957-506c5cbd7fa8" />